src/notation/qml/MuseScore/NotationScene/NotationToolBar.qml

```diff
@@ -85,6 +85,8 @@ Item {
             navigation.panel: root.navigationPanel
             navigation.name: toolTipTitle
             navigation.order: model.index
+            accessible.name: (item.checkable ? (item.checked ? item.title + "  " + qsTrc("global", "On") :
+                                                               item.title + "  " + qsTrc("global", "Off")) : item.text)
```

Comment:
Since `item` doesn't have a `text` property, this causes a QML warning to be printed when launching the app (`Unable to assign [undefined] to QString`). You probably meant `title` instead, didn't you?

Reply:
Here we are trying to address the `text` of item (MenuItem) - no property, in PlaybackToolBarActions we are trying to address the `text` of btn (FlatButton) - no problem
Probably we should use title in both cases

Reply:
@cbjeukendrup You're right. I will pay more attention to warnings and code structure from now on. Do I need to create a new pull request or something else to correct this?

Reply:
@shubham-shinde-442 
yes, please create a new pull request

Reply:
Sure